lib/sample_uploader/utils/importer.py

```diff
@@ -237,6 +247,11 @@ def _get_existing_sample(name, kbase_sample_id):
             e.row = row_num
             warnings.warn(e)
 
+    # remove samples in the existing_samples (input sample_set) but not in the input file
```

Comment:
should we add the ability to specify what kind of set operation to do on the input file and the provided existing input list?
this changes it to the `intersection` but maybe we should allow for the `union` to be utilized as well

Reply:
yea. That's a good idea. Maybe we can talk to the SMEs and do this in a new PR.